docs/space-desktop/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+date: "1"
+
+---
+The [Space Desktop App](https://github.com/FleekHQ/space-desktop){target=_blank} is available in the three most common operating systems: `macOS`, `Windows`, and `Linux`.
+
+You can download the right installer for you from our [Github repository latest release page](https://github.com/FleekHQ/space-desktop/releases/latest){target=_blank}.
+
+### macOS
+
+Download the `.dmg` image and drag the Space Desktop app to your Applications folder as you usually do with others `.dmg` apps.
+
+### Windows
+
+Download the `.exe` executable file and just install the Space Desktop app as you usually do with others Windows apps. 
```

Comment:
Is it common to install apps like this on Win? In case it's usually the MSI, maybe we can add a comment saying "Since the entire application is bundled into the exe, you can place that anywhere you'd like to store the app and add shortcuts as needed (e.g., your Taskbar or Desktop)."

Reply:
@jsonsivar yea, good point, what do u think about:

>Download the `.exe` executable file and execute the Space Desktop app as you usually do with other executable Windows apps. The entire application is bundled into the `exe`, you can place that anywhere you'd like to store the app and add shortcuts as needed (e.g., your Taskbar or Desktop).

Also, for the link on the release section, we can add it with a message that says:

> If you have any doubt or issue when installing the app, please check at our [Space Desktop app docs](https://docs.fleek.co/space-desktop/overview/)

Reply:
Both sound good 👍